applications/mne_x_libs/xDisp/helpers/realtimemultisamplearraymodel.cpp

```diff
@@ -406,7 +400,7 @@ void RealTimeMultiSampleArrayModel::addData(const QList<MatrixXd> &data)
     bool doComp = m_bCompActivated && m_matDataRaw.cols() > 0 && m_matDataRaw.rows() == m_matComp.cols() ? true : false;
 
     //SPHARA
-    bool doSphara = m_bSpharaActivated && m_matSpharaMultFirst.cols() > 0 && m_matSpharaMultFirst.rows() == m_matComp.cols() && m_matSpharaMultSecond.cols() > 0 && m_matSpharaMultSecond.rows() == m_matComp.cols() ? true : false;
+    bool doSphara = m_bSpharaActivated && m_matSparseSpharaMult.cols() > 0 && m_matDataRaw.rows() == m_matSparseSpharaMult.cols() ? true : false;
```

Comment:
don't use such complex expression in one line - prefer readability over compactness -> compiler is taking care for compressing everything